test-data/unit/check-classes.test

```diff
@@ -895,6 +895,43 @@ a.f.x # E: "int" has no attribute "x"
 [builtins fixtures/property.pyi]
 
 
+-- Descriptors
+-- -----------
+
+
+[case testAccessingNonDataDescriptor]
+import typing
+
+class D():
+    def __get__(self, instance: Any, owner: Any) -> str:
+        return 'some string'
+
+class A:
+    f = D()
+
+a = A() # type. A
```

Comment:
Oops. No, that's a typo. I think there are a couple of the Property tests w/ the same typo, so I'll fix them too.